### PR TITLE
Using timezone in maybeTick

### PR DESCRIPTION
**Pull request by theFong**
The `maybeTick` func which determines if a cuttoff should happen does not use the configured timezone. This results in incorrect behavior when the timezone and cuttoff time specified straddles a day since `IsWeekend` and `IsBankingDay` would use an incorrect date.

Related https://github.com/moov-io/base/pull/148

### CODE REVIEW

File: x/schedule/cutoff.go [hunk 1]
```diff
@@ -72,12 +72,17 @@ func (ct *CutoffTimes) register(tz string, timestamp string) error {
 	}
 
 	var zone string
+	var location *time.Location
 	if tz != "" {
 		zone = fmt.Sprintf("CRON_TZ=%s", tz)
+		l, _ := time.LoadLocation(tz)
+		location = l
+	} else {
+		location = time.UTC
```
Comment (vxio) [inline x/schedule/cutoff.go:81]: It'd be nice to have a log msg after this block to indicate which timezone we're using

**Review by vxio — APPROVED** [review]
Looks good to me

File: x/schedule/cutoff.go (hunk 1 above)
Comment (adamdecaf) [reply x/schedule/cutoff.go:81]: We can do that.

**Review by adamdecaf — APPROVED** [review]

File: x/schedule/cutoff.go (hunk 1 above)
Comment (adamdecaf) [reply x/schedule/cutoff.go:81]: https://github.com/moov-io/paygate/pull/636

**Pull request merged**